package.json

```diff
@@ -1,17 +1,14 @@
 {
     "name": "webknossos",
     "license": "AGPL-3.0",
-    "engines":
```

Comment:
in another pr, I also had the problem that the package.json was entirely reformatted. later I had annoying merge conflicts in other prs. could you try to undo these changes?
the proper solution will be to update biome and then let it run on package.json, too. the current version will always skip package.json (see https://github.com/biomejs/biome/issues/1668).
updating requires lots of small fixes, though, as biome got smarter. so, this should be a different pr.